triage_time_analysis.py

```diff
@@ -0,0 +1,94 @@
+from typing import List, Optional
+import matplotlib.pyplot as plt
+import pandas as pd
+from datetime import timedelta
+
+from data_loader import DataLoader
+from model import Issue, Event
+import config
+
+class TriageTimeAnalysis:
+    def __init__(self):
+        self.USER: Optional[str] = config.get_parameter("user")
+
+    def run(self):
+        issues: List[Issue] = DataLoader().get_issues()
+        total_events = sum(len([e for e in issue.events if self.USER is None or e.author == self.USER]) for issue in issues)
+        output = f"Found {total_events} events across {len(issues)} issues"
+        if self.USER is not None:
+            output += f" for {self.USER}."
+        else:
+            output += "."
+        print("\n\n" + output + "\n\n")
+
+    def _first_assignment_event(self, issue: Issue) -> Optional[Event]:
+        created = issue.created_date
+        if created is None:
+            return None
+        sorted_events = sorted(issue.events, key=lambda ev: ev.event_date or created)
+        for e in sorted_events:
+            if not getattr(e, "event_type", None):
+                continue
+            et = str(e.event_type).strip().lower()
```

Comment:
e.event_type will already be a lowercase stripped string
```suggestion
            et = e.event_type
```

Reply:
Updated code based on feedback